spec/support/migrations/1_create_link_table.rb

```diff
@@ -8,6 +8,8 @@ def up
       t.string :normal_target_type
       t.integer :source_id
       t.integer :source_type
+
+      t.references :legacy_source, polymorphic: true
```

Comment:
Maybe we should add a new migration instead of adding this to an existing migration? Adding this here requires library maintainers to run a `bin/rake db:reset` to get their DBs in the appropriate state.